packages/venia-concept/src/components/Checkout/Receipt/receipt.js

```diff
@@ -62,7 +62,7 @@ class Receipt extends Component {
                     </div>
                     <Button
                         data-id={CONTINUE_SHOPPING_BUTTON_ID}
-                        classes={darkThemeClasses}
+                        classes={{ root: classes.button }}
```

Comment:
I'm not sure what to do about it but this is a bit clunky.

Reply:
Basically, this is an idea of our classify hoc 🙂, it allows us to change styles for some elements by passing them as a `classes` object to the component.

In general, I noticed that we added a new feature to classify hoc, so now it is possible to write something like this:

```
<Button className={classes.button}></Button>
```